PowerUp/app/src/main/java/powerup/systers/com/CompletedScene.java

```diff
@@ -20,8 +20,16 @@ public void onClick(View v) {
 				Intent myIntent = new Intent(CompletedScene.this,
 						MapActivity.class);
 				startActivityForResult(myIntent, 0);
+				finish();
 			}
 		});
 	}
-	
+
+	@Override
+	public void onBackPressed() {
+		super.onBackPressed();
+		Intent i = new Intent(CompletedScene.this,MapActivity.class);
```

Comment:
Create a better name for the Intent. Something like `intent`. You could also declare it in line when you call `startActivity`